src/cmd/cli/command/compose.go

```diff
@@ -168,7 +165,12 @@ func makeComposeUpCmd() *cobra.Command {
 				if errors.As(context.Cause(tailCtx), &errDeploymentFailed) {
 					// Tail got canceled because of deployment failure: prompt to show the debugger
 					term.Warn(errDeploymentFailed)
-					interactiveAIDebug(cmd.Context(), errDeploymentFailed, deploy, project)
+					if !nonInteractive {
+						failedServices := []string{errDeploymentFailed.Service}
+						track.Evt("Debug Prompted", P("failedServices", failedServices), P("etag", deploy.Etag), P("reason", errDeploymentFailed))
+						// Call the AI debug endpoint using the original command context (not the tailCtx which is canceled)
+						_ = cli.InteractiveDebug(cmd.Context(), provider, deploy.Etag, project, failedServices)
```

Comment:
Why bother to assign to `_` and return the ErrDebugSkipped?

Reply:
I wanted `Debug` and `cli.InteractiveDebug` to have the same signature.